aviary/subsystems/propulsion/engine_deck.py

```diff
@@ -1252,6 +1258,22 @@ def _count_data(self):
         self.data_indices = data_indices.astype(int)
 
 
+class TurboPropDeck(EngineDeck):
```

Comment:
Since we're planning to extend this to use the prop model, it is no longer "just" a deck. I suggest the more general `TurboProp` for the class name

Reply:
I think it's worth noting that this is a deck based turboprop vs pycycle, NPSS, etc

Reply:
Yeah that's fair. Let's keep the name then!